moped-database/migrations/1678894870999_geo_constraints/down.sql

```diff
@@ -0,0 +1,2 @@
+-- see original table declarations in 1666978291816_add_component_geometry_tables/up.sql
+SELECT 0;
```

Comment:
i got a little lazy here. i can write the migrations if y'all say so 🙈

Reply:
😬